tests/scripts/check-scripts

```diff
@@ -72,7 +72,7 @@ main()
     SERVICES_PID=$!
     sudo service tayga start
     echo 'Waiting for services to be ready...'
-    sleep 10
+    sleep 30
```

Comment:
Thanks @jwhui ! Ididn't really get a chance to reproduce the failure in my docker environment perhaps because I was testing on the workstation which should be much more powerful than the GitHub Action virtual machine. But it is still surprising that it takes 30 seconds to start the tayga service :)

I didn't rerun the checks for another 2 times and it passes consistently so I think this should be the fix, Thanks!

Reply:
The problem is actually waiting for `./script/console` to launch all the services. `check_otbr_agent` below fails when not all the OTBR services have been started yet.